module_24_Python/4_rpg_game.py

```diff
@@ -0,0 +1,130 @@
+"""
```

Comment:
Create a separate directory for this task and place other used files there

Reply:
created